### PR TITLE
random characters added

### PR DESCRIPTION
**Pull request by adprojectsstudio**
enhanchments #8

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ﻿// See https://aka.ms/new-console-template for more information
 Console.WriteLine("Hello, World!");
 
+Console.WriteLine("adsskjnmfdslfnsdlfnsdlfnsdlkfn");
```
Comment (angelo-p82) [inline Program.cs:4]: Add a point at the end

**Review by angelo-p82 — CHANGES_REQUESTED** [review]

File: Program.cs [hunk 2]
```diff
@@ -1,4 +1,5 @@
 ﻿// See https://aka.ms/new-console-template for more information
 Console.WriteLine("Hello, World!");
```
Comment (angelo-p82) [inline Program.cs:2]: Remove ! at the end

**Review by angelo-p82 — CHANGES_REQUESTED** [review]

File: Program.cs (hunk 2 above)
Comment (adprojectsstudio) [reply Program.cs:2]: it is as i say

Comment (adprojectsstudio) [reply Program.cs:2]: test comment

Comment (angelo-p82) [reply Program.cs:2]: hello

File: Program.cs [hunk 3]
```diff
@@ -1,4 +1,5 @@
...
 
+Console.WriteLine("adsskjnmfdslfnsdlfnsdlfnsdlkfn");
 Console.WriteLine("Press ENTER key");
```
Comment (angelo-p82) [inline Program.cs:5]: Add a ReadKey

**Review by angelo-p82 — CHANGES_REQUESTED** [review]

**Review by angelo-p82 — APPROVED** [review]

**Pull request merged**